src/main/cpp/t04_both2.cpp

```diff
@@ -23,9 +23,39 @@
 
 #include "t04_both2.h"
 #include <iostream>
+#include <set>
+#include <algorithm>
+#include <iterator>
 
 using namespace std;
 
 int t04_both2() {
 
+
+	set <int> a;
+	set <int> a1;
+	set <int> s3;
+	int c;
+	cin >> c;
+
+	for (int i = 0; i < c; i++)
+	{
+		int b;
+		cin >> b;
+		a.insert(b);
+	}
+	int c1;
+	cin >> c1;
+	int k = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'k' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4951848)